projects/stackoverflow/ExtractedRow.java

```diff
@@ -0,0 +1,57 @@
+package projects.stackoverflow;
+
+public class ExtractedRow {

```

Comment:
I suggest that instead of this class, you use a proto (Protocol Buffers).
You can see an example of a proto here:
https://github.com/hasadna/hasadna/blob/master/projects/opentrain/gtfs_pipeline/common/common.proto

It has getters and setters, and also a `writeDelimitedTo` that writes directly to a file stream:
https://github.com/hasadna/hasadna/blob/master/projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java#L56

So using it, you also won't need `writeExtractedRowToOutput()` and `writeEndOfLine()`.

You can also later read them from the file directly to an object.

You'll need to define 2 build targets:
https://github.com/hasadna/hasadna/blob/master/projects/opentrain/gtfs_pipeline/common/BUILD

Then, you can depend on them from your java file to get the proto imports:
https://github.com/hasadna/hasadna/blob/master/projects/opentrain/gtfs_pipeline/data_handler/BUILD#L13

If you prefer, I can approve this PR and you do it in a subsequent PR.